sensor_fusion/src/wgs84_utils/wgs84_utils.cpp

```diff
@@ -155,6 +141,43 @@ tf2::Vector3 wgs84_utils::geodesic_to_ecef(const wgs84_utils::wgs84_coordinate &
     return point_in_ned;
 }
 
+  /**
+   * Converts a given 3d cartesian point in the ECEF frame into a WSG-84 geodesic location
+   * The provided point should be defined relative to a frame which has a transform with the ECEF
+   * @param point The cartesian point to be converted
+   * @return The calculated WSG-84 geodesic location
+   */
+  wgs84_utils::wgs84_coordinate wgs84_utils::ecef_to_geodesic(const tf2::Vector3& point) {
```

Comment:
Can you add a simple unit test for this to make sure the math is correct?

Reply:
Just realize you said it is from geometry package. So I assume it should be correct. Maybe just copy the unit test over as well?

Reply:
Yeah good idea

Reply:
Removed for now as function is unused per our dicsussion